### DISCUSSION TITLE
ElastAlert: appended to my alerts

### DISCUSSION THREAD

**Discussion by BlakeRash** [jertel/elastalert2#581] [Q&A] [answered]
I was wanting to know if there is a way to remove the ElastAlert: from the beginning of my alerts? 
![image](https://user-images.githubusercontent.com/95305157/144092860-28786d4f-c525-45be-adc9-d0c985a9e293.png)
Is there a configuration issue I have missed when setting this up? 

Thank you!

**Comment by jertel** [comment, marked as answer]
Yes. This is covered in the [documentation](https://elastalert2.readthedocs.io/en/latest/ruletypes.html?highlight=alert_subject#alert-subject).

**Reply by BlakeRash** [reply]
Thank you sir! Much appreciated.
